smplpytorch/pytorch/smpl_layer.py

```diff
@@ -149,6 +149,11 @@ def forward(self,
                 th_jtr = th_jtr - center_joint
                 th_verts = th_verts - center_joint
         else:
+            if self.center_idx is not None:
```

Comment:
Seems like this if block (and the one above in line 147-150) is independent of the surrounding clauses.